lib/ansible/utils/color.py

```diff
@@ -86,10 +86,10 @@ def colorize(lead, num, color):
 def hostcolor(host, stats, color=True):
     if ANSIBLE_COLOR and color:
         if stats['failures'] != 0 or stats['unreachable'] != 0:
-            return u"%-37s" % stringc(host, 'red')
+            return u"%-37s" % stringc(host, COLOR_ERROR)
         elif stats['changed'] != 0:
-            return u"%-37s" % stringc(host, 'yellow')
+            return u"%-37s" % stringc(host, COLOR_CHANGED)
         else:
-            return u"%-37s" % stringc(host, 'green')
+            return u"%-37s" % stringc(host, COLOR_OK)
```

Comment:
Apologies.  They come from constants, however I forgot to refer to this.
The pull request should now be updated.

Without this change, the hostname will always be displayed in 1 of the 3 hard coded colours.  This should allows them to match whatever has been specified in the config.

Thanks.